docs/editors-guide/project-management.md

```diff
@@ -0,0 +1,45 @@
+# Mondo Project management
+
+## Description
+
+Documentation of project management tasks in Mondo.
+
+## Recurring meetings
+Mondo meetings should be added to the Monarch shared google calendar. Members of the curation and technical teams are outlined [here](https://mondo.monarchinitiative.org/pages/contributors/).
+
+- **Curation call**: Thursdays, 9am PT
+- **Technical call**: Fridays, 10am PT
+- **Quarterly Medical Expert calls**: Next meeting is Thursday, Dec 01, 12-1pm PT. Future calls will be scheduled.
+
+## Mondo workshops
+- Any upcoming workshops should advertised 
+  - on our [website](https://mondo.monarchinitiative.org/pages/workshop/)
+  - on our Slack channel
```

Comment:
add url for slack channel and mailing list here? or are you worried about exposing those too publicly?

Reply:
I don't think we want to add those here- this is a guide for project managers and they should know those things.